chainconfig/chainconfig_routine.go

```
@@ -90,3 +90,16 @@ func (cc *ChainConfigRoutine) run() {
 		time.Sleep(time.Duration(cc.cfg.EnableFeatureInterval) * time.Second)
 	}
 }
+
+func (cc *ChainConfigRoutine) init() {
+	if cc.node.IsValidator() {
+		dappClient := client.NewDAppChainRPCClient(cc.chainID, cc.cfg.DAppChainWriteURI, cc.cfg.DAppChainReadURI)
+		chainConfigClient, err := NewChainConfigClient(dappClient, cc.address, cc.signer, cc.logger)
+		if err != nil {
+			cc.logger.Error("Failed to create ChainConfigClient", "err", err)
+		} else {
+			// NOTE: errors are logged by the client, no need to log again
+			_ = chainConfigClient.SetBuildNumber(cc.buildNumber)
```

if error is ignored, we can just do
```
chainConfigClient.SetBuildNumber(cc.buildNumber)
```